### PR TITLE
intial commit

### PR DESCRIPTION
**Pull request by Vivek20ingh**

### CODE REVIEW

File: Components/Movielist.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+const list = [
```
Comment (thehellmaker) [inline Components/Movielist.js:1]: .json and .js files are duplicate. Why do you need both?

File: Components/Search.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  const ItemView = ({item}) =>{
+    return(
+     
+     <ListItem>
+     <Avatar source={{uri: item.avatar_url}} />
+       <ListItem.Content>
+       <ListItem.Title>{item.title}</ListItem.Title>
+       <ListItem.Subtitle>{item.year}</ListItem.Subtitle>
+     </ListItem.Content>
+     </ListItem>
```
Comment (thehellmaker) [inline Components/Search.js:52]: list item is duplicated in both search and list page. This duplication is a bad idea considering they both are using the same view

**Review by thehellmaker — CHANGES_REQUESTED** [review]

File: Components/Movielist.js (hunk 1 above)
Comment (Vivek20ingh) [reply Components/Movielist.js:1]: I was trying some different method. I have deleted .json file.

File: Components/Search.js (hunk 2 above)
Comment (Vivek20ingh) [reply Components/Search.js:52]: I will merge both of them through one function

**Pull request merged**